### PR TITLE
Update init-job

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

Related-to: https://github.com/thoth-station/init-job/issues/345

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Refactor init-job into initializer workflow

### CODE REVIEW

File: init-job/base/argo-workflows/initialize-schema.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: initialize-schema
+  annotations:
```
Comment (goern) [inline init-job/base/argo-workflows/initialize-schema.yaml:6]: we dont need these labels and annotations

File: init-job/base/argo-workflows/initialize-schema.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    thoth-station.ninja/template-version: 0.1.0
+  labels:
+    app: thoth
+    component: initialize-schema
+spec:
+  templates:
+    - name: initialize-schema
+      inputs:
+        parameters:
```
Comment (goern) [inline init-job/base/argo-workflows/initialize-schema.yaml:15]: we dont need these parameters, we need to make sure that the imagestream `init-job` is created where the workflow runs. I think that will be middletier?

Comment (pacospace) [reply init-job/base/argo-workflows/initialize-schema.yaml:15]: yes exactly

Comment (pacospace) [reply init-job/base/argo-workflows/initialize-schema.yaml:15]:
```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: stage-thoth-init-job
spec:
  destination:
    namespace: thoth-middletier-stage
    server: "https://api.thoth01.lab.upshift.rdu2.redhat.com:6443"
  source:
    path: adviser/overlays/stage
    repoURL: "https://github.com/thoth-station/thoth-application.git"
    targetRevision: master
  project: thoth
  syncPolicy:
    automated:
      automated:
        prune: false
        selfHeal: false
      selfHeal: true
```

**Pull request merged**